Blockchain - Improve First Workout (#2100)
enkidevs/curriculum: blockchain/fundamentals/intro-blockchain/importance-of-blockchain.md

---
author: mihaiberq

type: normal

category: must-know

aspects:
  - introduction

links:
  - '[Examples of Blockchain Use-Cases](https://medium.com/@matteozago50-examples-of-how-blockchains-are-taking-over-the-world-4276bf488a4b){website}'

---

# Why Should You Care?

---
## Content
           	
Here are a couple reasons on why blockchain is an important technology:

1. Secure

> All data found in the blockchain is secured using advanced cryptography. This makes it resistant to unauthorized changes (like hacking).

Reviewer: This shows up indented in the app and looks weird. 

Let's add a new line?

```suggestion

All data found in the blockchain is secured using advanced cryptography. This makes it resistant to unauthorized changes (like hacking).
```

> Because the blockchains is decentralized, it is less susceptible to human error, data loss or hacking.

Reviewer: ```suggestion
Because the blockchain is decentralized, it doesn't have a single point of failure. This makes it less susceptible to abuse, human error, data loss or hacking.
```

> 2. Transparent
> The blockchain contains a full history of transactions, not just a snapshot in time. Moreover, the same history is spread across all blocks of the network, with all blocks storing **the same information**.

Reviewer: ```suggestion
2. Transparent

The blockchain contains a full history of transactions, not just a snapshot in time. Moreover, the same history is spread across all blocks of the network, with all blocks storing **the same information**.
```

> Another proof of the worthiness and capabilities of this technology comes from Estonia. The small european country started managing its registers, such as national health, judicial, security and commercial code systems - on their own blockchain. The system has been live since 2012 and their initiative is only moving forward.

Reviewer: ```suggestion
Another proof of the worthiness and capabilities of this technology comes from Estonia. The small european country started managing its registers, such as national health, judicial, security and commercial code systems - on their own blockchain. The system has been live since 2012 and their initiative is only moving forward.
```
```suggestion
Another proof of the worthiness and capabilities of this technology comes from Estonia. 

The small European country started managing its registers, such as national health, judicial, security and commercial code systems - on their own blockchain. The system has been live since 2012 and their initiative is only moving forward.
```

Reviewer: We should also add a "Learn More" link about Estonia's use of blockchain

And maybe a good article that shows all real world uses today?

Reviewer: [The current link gives a 404](https://medium.com/@matteozago50-examples-of-how-blockchains-are-taking-over-the-world-4276bf488a4b)

Author: hmm that is weird, I thought I used [this url](https://medium.com/@matteozago/50-examples-of-how-blockchains-are-taking-over-the-world-4276bf488a4b) which worked. i'll update
